### PR TITLE
PPU Compilation: Search for more game data directories.

### PR DESCRIPTION
**Pull request by Megamouse**
Naive approach.
- Search for more game data directories for PPU compilation
- Also properly remove duplicate paths.
- Set thread priority to low for GUI updates.

TODO:
- [x] Add similar code to "Create PPU Cache".

### CODE REVIEW

File: rpcs3/Emu/Cell/PPUThread.cpp [hunk 1]
```diff
@@ -2069,7 +2070,13 @@ extern void ppu_finalize(const ppu_module& info)
 
 extern void ppu_precompile(std::vector<std::string>& dir_queue, std::vector<lv2_prx*>* loaded_prx)
 {
-	// Remove duplicates
+	// Make sure we only have one '/' at the end and remove duplicates.
+	for (std::string& dir : dir_queue)
+	{
+		while (dir.back() == '/' || dir.back() == '\\')
+			dir.pop_back();
```
Comment (elad335) [inline rpcs3/Emu/Cell/PPUThread.cpp:2077]: Maybe try dir.erase(dir.begin() + (dir.find_last_not_of(fs::delim) +1), dir.end());

Comment (Megamouse) [reply rpcs3/Emu/Cell/PPUThread.cpp:2077]: This doesn't make much sense since we already manually add /

**Comment by Megamouse** [comment]
Added a unified function that should also work with Create PPU Cache

**Comment by Megamouse** [comment]
Setting thread priority to low for GUI updates

File: rpcs3/Emu/Cell/PPUThread.cpp (hunk 1 above)
Comment (elad335) [reply rpcs3/Emu/Cell/PPUThread.cpp:2077]: Its about not calling pop_back potentially many times.

Comment (Megamouse) [reply rpcs3/Emu/Cell/PPUThread.cpp:2077]: I can do it twice for / and \.
But what's the big difference for a couple of dirs

**Pull request merged**